web/react/components/get_link_modal.jsx

```diff
@@ -9,6 +9,7 @@ ZeroClipboardMixin.ZeroClipboard.config({
 });
 
 module.exports = React.createClass({
+    displayName: 'GetLinkModal',
```

Comment:
This is React's nice name for the class. It appears in the react debugger. It is required by the style guide. When we move to using the ES6 class syntax I don't think this will be required anymore.